lib/qesapdeployment.pm

```diff
@@ -1133,6 +1134,37 @@ sub qesap_ansible_fetch_file {
     return $args{out_path} . $args{file};
 }
 
+=head3 qesap_ansible_reg_module
+
+    Compose the ansible-playbook argument for the registration.yaml playbook,
+    about an additional module registration
+
+    -e sles_modules='[{"key":"SLES-LTSS-Extended-Security/12.5/x86_64","value":"*******"}]'
+
+    Known limitation is that registration.yaml support multiple modules to be registered,
+    this code only support one.
+
+=over
+=item B<reg> - name and reg_code for the additional extension to register.
+                This argument is a two element comma separated list string.
+                Like: 'SLES-LTSS-Extended-Security/12.5/x86_64,123456789'
+                First string before the comma has to be a valid SCC extension name, later used by Ansible
+                as argument for SUSEConnect or registercloudguest argument.
+                Second string has to be valid registration code for the particular extension.
+
+=back
+=cut
+
+sub qesap_ansible_reg_module {
+    my (%args) = @_;
+    croak 'Missing mandatory "reg" argument' unless $args{reg};
+    my @reg_args = split(/,/, $args{reg});
+    die "Missing reg_code for '$reg_args[0]'" if scalar @reg_args != 2;
```

Comment:
```suggestion
    die "Missing reg_code for '$reg_args[0]'" if (@reg_args != 2);
```
`scalar` keyword is not needed. `!= 2` will make Perl treat `@reg_args` as an scalar.